lib/cdo/shared_constants/pd/shared_workshop_constants.rb

```diff
@@ -72,6 +73,18 @@ module SharedWorkshopConstants
         SUBJECT_CSP_FIT = SUBJECT_FIT,
         SUBJECT_CSP_FOR_RETURNING_TEACHERS,
       ],
+      COURSE_CSA => [
+        SUBJECT_CSA_SUMMER_WORKSHOP = SUBJECT_SUMMER_WORKSHOP,
+        SUBJECT_CSA_VIRTUAL_KICKOFF = SUBJECT_VIRTUAL_KICKOFF,
```

Comment:
Virtual Kickoff shows up here but not in the other places that this list shows up. Is that an issue?

Reply:
Which other places? I see that we have it for csd and csp, this list was intended to match csp (minus returning teachers workshop)

Reply:
workshop_survey_constants.rb and workshop_constants.rb

Reply:
did some digging and asking of ben, and it looks like the survey constants ("CATEGORY_MAP" "VALID_DAYS", etc) are used by the jotform survey code, which is the predecessor to the foorm survey stuff. I'm not sure where/if the logic that would replace this specific stuff exists, but that would explain why it might be out of date (the virtual stuff was added more recently, maybe fit too?) according to ben: We were intending to remove all the jotform related code about a year after we stopped collecting any data using it, which would be this fall sometime.

as far as the TIME_CONSTRAINTS, that looks like its used by payments related code to figure out how much to charge for workshops, which might not be relevant for virtual kickoff and fit workshops? I'm assuming if payment related stuff was off, someone would notice pretty quick, but i'll double check with tess.

Reply:
ha, according to tess/tawny the payments related code is not currently in use either